persistence/sql/persister_oauth2.go

```diff
@@ -533,23 +554,7 @@ func (p *Persister) DeletePKCERequestSession(ctx context.Context, signature stri
 func (p *Persister) RevokeRefreshToken(ctx context.Context, id string) (err error) {
 	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.RevokeRefreshToken")
 	defer otelx.End(span, &err)
-	return p.deactivateSessionByRequestID(ctx, id, sqlTableRefresh)
-}
-
-func (p *Persister) RevokeRefreshTokenMaybeGracePeriod(ctx context.Context, id string, _ string) (err error) {
-	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.RevokeRefreshTokenMaybeGracePeriod")
-	defer otelx.End(span, &err)
-
-	/* #nosec G201 table is static */
-	return sqlcon.HandleError(
-		p.Connection(ctx).
-			RawQuery(
-				fmt.Sprintf("UPDATE %s SET active=false, first_used_at = CURRENT_TIMESTAMP WHERE request_id=? AND nid = ? AND active", OAuth2RequestSQL{Table: sqlTableRefresh}.TableName()),
-				id,
-				p.NetworkID(ctx),
-			).
-			Exec(),
-	)
+	return p.deleteSessionByRequestID(ctx, id, sqlTableRefresh)
```

Comment:
Revoking a token should completely invalidate it. So now, we delete it. Otherwise, it could trigger graceful token rotation detection potentially.